docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<p>Do we support <code>toshiba_bifurcation_qpu</code> with dry-run currently? I couldn't run this line yet --perhaps I'm missing context from another PR.</p><p> </p><p>Curious aside: Without dry-run enabled, are you able to submit to <code>toshiba_bifurcation_qpu</code> without a user's own Azure credentials? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>now that https://github.com/Infleqtion/server-superstaq/pull/2397 is merged, we should be able to support both for <code>toshiba_bifurcation_qpu</code> with and without the <code>dry-run</code>. </p><p> </p><p>I believe users need to set their own Azure credentials to submit to <code>toshiba_bifurcation_qpu</code> without dry-run enabled.</p>

Reply:
> Do we support toshiba_bifurcation_qpu with dry-run currently? 

This should work once @dhirpalshah deploys https://github.com/Infleqtion/server-superstaq/pull/2432

> are you able to submit to toshiba_bifurcation_qpu without a user's own Azure credentials? 

Yes.

Reply:
Ah I see, thanks!